src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_SummaryScreen/TicketSummary/use-ticket-summary.ts

```diff
@@ -0,0 +1,82 @@
+import {useTicketAssistantState} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistantContext';
+import {
+  calculateSavings,
+  calculateSingleTickets,
+  getIndexOfLongestDurationTicket,
+  perTripSavings,
+} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_SummaryScreen/TicketSummary/utils';
+import {getReferenceDataName} from '@atb/reference-data/utils';
+import {TicketAssistantTexts, useTranslation} from '@atb/translations';
+
+export const useTicketSummary = () => {
+  const {t, language} = useTranslation();
+  const {response, purchaseDetails, data} = useTicketAssistantState();
+  const {tickets, zones, total_cost, single_ticket_price} = response;
+  const {frequency} = data;
+  const index = getIndexOfLongestDurationTicket(tickets);
+  const ticket = tickets[index];
+  const recommendedTicketTypeConfig =
+    purchaseDetails?.purchaseTicketDetails[index]?.fareProductTypeConfig;
+  const preassignedFareProduct =
+    purchaseDetails?.purchaseTicketDetails[index]?.preassignedFareProduct;
```

Comment:
Same question here regarding using `index` instead of an id as identifier across arrays.